erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -305,8 +306,8 @@ def _valid_for_reserve(item_code, warehouse):
 				"reserved_qty": get_reserved_qty(item_code, warehouse)
 			})
 
-	def on_update(self):
-		pass
```

Comment:
Keep this, in case it's being used somwhere

Reply:
same thing i guess required for all files?

Reply:
Don't remove it from the other files atleast, you don't need to add it